tests/integration/test_stages.py

```diff
@@ -1,3 +1,4 @@
+# pylint:disable=protected-access,no-self-use
```

Comment:
@e-kolpakov This was copied from other test files, for me rationale is that in in tests we might need to check for "private" variables often, as well as many mock overrides will not use "self". 

Let me know if I convinced you.